tasks/Task.cpp

```diff
@@ -4,6 +4,7 @@
 #include <iodrivers_base/ConfigureGuard.hpp>
 #include <genset_whisperpower_ddc/VariableSpeed.hpp>
 #include <genset_whisperpower_ddc/ControlCommand.hpp>
+#include <fstream>
```

Comment:
Also leftover because of the cout ? You only need iostream for cout

Reply:
Sorry, should have paid more attention. It was actually for a workaround I did before you showed me the correct location of the log file.